plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -832,6 +832,10 @@ class read_write {
 
 } // namespace chain_apis
 
+struct chain_plugin_metrics {
+   //runtime_metric
```

Comment:
I assume this will be cleaned up with the chain_plugin PR to the feature branch.